src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -35,7 +35,10 @@ const SUPPORTED_FIAT_ACCOUNT_SCHEMAS = new Set<FiatAccountSchema>([
   FiatAccountSchema.MobileMoney,
   FiatAccountSchema.PIXAccount,
 ])
-const SUPPORTED_KYC_SCHEMAS = new Set<KycSchema>([KycSchema.PersonalDataAndDocuments])
+const SUPPORTED_KYC_SCHEMAS = new Set<KycSchema>([
+  KycSchema.PersonalDataAndDocuments,
+  KycSchema.PersonalDataAndDocumentsDetailed,
```

Comment:
since no actual client side changes were needed, I wonder if we should switch to using remote config for supported kyc schemas. it makes sense to keep fiat account schemas in code since code changes are actually needed to add support for those.